src/rpc/net.cpp

```diff
@@ -31,7 +31,7 @@
 
 const std::vector<std::string> CONNECTION_TYPE_DOC{
         "outbound-full-relay (default automatic connections)",
-        "block-relay-only (does not relay transactions or addresses)",
+        "outbound-block-relay (does not relay transactions or addresses)",
```

Comment:
This might be easier to confuse with peers that request blocks only (#20726)

Reply:
Yes, the script can be updated depending on what is decided there.